libcontainer/init_linux.go

```diff
@@ -494,6 +494,11 @@ func signalAllProcesses(m cgroups.Manager, s os.Signal) error {
 		logrus.Warn(err)
 	}
 
+	subreaper, err := system.GetSubreaper()
+	if err != nil {
+		return err
```

Comment:
One thing, this will fail on kernels older than 3.4 that don't support it.  I would maybe ignore the error and set subreaper to `0` in this case.

Reply:
Good point. We don't want to fail in such a case since the application is still supposed to work.

Reply:
@crosbymichael I think I should make this a bit smarter, by assigning `subreaper = 0` only when the error is `EINVAL`. WDYT ?

Reply:
No nevermind, in case of `PR_GET_CHILD_SUBREAPER`, there is no specific errno set.

Reply:
Sounds good but I don't know what returning this error here would accomplish in the execution flow.  Either works for me.

Reply:
I have updated the PR with the simple case. We fallback to original behavior in case getting subreaper status fails.